webpack.config.js

```diff
@@ -40,7 +40,7 @@ module.exports = {
   },
   output: {
     filename: 'moment-range.js',
-    library: 'moment-range',
+    library: 'MomentRange',
```

Comment:
so, this'll change the name for all module types. ideally i'd like to keep it as `moment-range` everywhere except the browser